notebooks/Liquidators_data.ipynb

```diff
@@ -0,0 +1,1325 @@
+    "        for protocol, url in config.items():\n",
+    "            print(f\"Processing {protocol} from Google Storage...\")\n",
+    "            df_protocol = load_data_from_google_storage(url)\n",
+    "            df_protocol['Protocol'] = protocol\n",
+    "            dataframes_protocol[protocol] = df_protocol\n",
+    "    elif source == 'local_parquet':\n",
+    "        for protocol, file_path in config.items():\n",
+    "            print(f\"Processing {protocol} from local Parquet file...\")\n",
+    "            df_protocol = load_data_from_local_parquet(file_path)\n",
+    "            df_protocol['Protocol'] = protocol\n",
+    "            dataframes_protocol[protocol] = df_protocol\n",
+    "    return dataframes_protocol\n",
+    "\n",
+    "# Configuration for Google Storage\n",
+    "google_storage_config = {\n",
+    "    'zklend': 'https://storage.googleapis.com/derisk-persistent-state/zklend_data/events_sample.parquet',\n",
+    "    'nostra_alpha': 'https://storage.googleapis.com/derisk-persistent-state/nostra_alpha_data/events_sample.parquet',\n",
+    "    'nostra_mainnet': 'https://storage.googleapis.com/derisk-persistent-state/nostra_mainnet_data/events_sample.parquet',\n",
+    "    'hashstack_v0': 'https://storage.googleapis.com/derisk-persistent-state/hashstack_v0_data/events_sample.parquet',\n",
+    "    'hashstack_v1': 'https://storage.googleapis.com/derisk-persistent-state/hashstack_v1_data/events_sample.parquet'\n",
+    "}\n",
+    "\n",
+    "# Configuration for local Parquet files\n",
+    "local_parquet_config = {\n",
+    "    'zklend': 'path_to_local_parquet_files/zklend_data/events_sample.parquet',\n",
+    "    'nostra_alpha': 'path_to_local_parquet_files/nostra_alpha_data/events_sample.parquet',\n",
+    "    'nostra_mainnet': 'path_to_local_parquet_files/nostra_mainnet_data/events_sample.parquet',\n",
+    "    'hashstack_v0': 'path_to_local_parquet_files/hashstack_v0_data/events_sample.parquet',\n",
+    "    'hashstack_v1': 'path_to_local_parquet_files/hashstack_v1_data/events_sample.parquet'\n",
+    "}\n",
+    "\n",
+    "# Select the source: 'google_storage' or 'local_parquet'\n",
+    "source = 'google_storage'  # or 'local_parquet'\n",
+    "config = google_storage_config if source == 'google_storage' else local_parquet_config\n",
+    "\n",
+    "# Load the data\n",
+    "dataframes_protocol = load_protocol_data(source, config)\n",
+    "\n",
+    "# dataframes_protocol\n",
+    "for protocol, df in dataframes_protocol.items():\n",
+    "    print(f\"{protocol} dataframe shape: {df.shape}\")\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 323,
+   "id": "176c3fe0",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# 1. ZKLEND\n",
+    "\n",
+    "# Liquidation events for the zklend protocol\n",
+    "filtered_df_zklend = dataframes_protocol['zklend'].loc[dataframes_protocol['zklend'][\"key_name\"] == 'Liquidation']\n",
```

Comment:
I prefer if you combine the individual dataframes to one big dataframe (and add a `protocol` column) instead of keeping them separate with a dict.